TaskManager/client/client.js

```diff
@@ -1,7 +1,11 @@
-let tasks = null; //JSON.parse(localStorage.getItem("tasks")) || [];
-const tasks_url = "http://localhost:5000/tasks";
-let completedTasks = JSON.parse(localStorage.getItem("completedTasks")) || [];
+let tasks = [];
+let completedTasks = [];
 let firstRun = true;
+const routes = {
+	tasks: "http://localhost:5000/tasks",
+	finishTask: "http://localhost:5000/finishTask",
+	login: "http://localhost:5000/login",
```

Comment:
It's best to not explicitly state the domain, so it'll work wherever you run it, locally or online
```suggestion
	tasks: "/tasks",
	finishTask: "/finishTask",
	login: "/login",
```